libraries/libopaeuio/opaeuio.c

```diff
@@ -338,7 +353,9 @@ STATIC int opae_uio_init(struct opae_uio *u, const char *dfl_device)
 out_destroy:
 	opae_uio_destroy(u);
 out_glob_free:
-	opae_globfree(&globbuf);
+	if (glob_res || !globbuf.gl_pathc) {
```

Comment:
Are these checks necessary / correct?

Reply:
if user pass uio, with out this checks sig fault

if user pass dfl_dev.0 , no sig fault